java/pom.xml

```diff
@@ -486,11 +487,6 @@
 
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>com.google.flatbuffers</groupId>
-        <artifactId>flatbuffers-java</artifactId>
-        <version>${dep.fbs.version}</version>
-      </dependency>
```

Comment:
Since this is in the dependency management section, couldn't it stay here?

Reply:
yes, I misunderstood what this section does.